Project/Assets/Editor/Twine Importer/TwineImporterUI.cs

```diff
@@ -42,43 +31,40 @@ void OnGUI ()
 	{
 		GUILayout.Label ("Import Twine Data", EditorStyles.boldLabel);
 
+		GUILayout.Label ("Twine JSON text:");
+	
+		this.jsonString = GUILayout.TextArea (this.jsonString, GUILayout.MinHeight(20), GUILayout.MaxHeight(50));
```

Comment:
My big worry with presenting a big empty text area is that it _implies_ that users are expected to type in JSON.

I understand the new exporter doesn't add to a file, but providing a import from file options, a placeholder that says "paste in JSON from ....", or even a button that automatically replaces the contents of the text area with the clipboard, would all go a long way to making it clear that users _should under no circumstances be writing out JSON_.